pipeline-examples/parallel-multiple-nodes/ParallelMultipleNodes.groovy

```diff
@@ -0,0 +1,12 @@
+def labels = ['precise', 'trusty'] // labels for Jenkins node types we will build on
+def builders = [:]
+for (x in labels) {
```

Comment:
Is there a reason to not to define this as `for (label in labels) {` and skip the `def label = x` below?